### PR TITLE
deleted things

### PR DESCRIPTION
**Pull request by rchubrich**
Hi Jackie! I deleted things.

### CODE REVIEW

File: lib/twitter/source_user.rb [hunk 1]
```diff
@@ -1,8 +1,5 @@
 require 'twitter/basic_user'
 
 module Twitter
-  class SourceUser < Twitter::BasicUser
```
Comment (jackieiscool) [inline lib/twitter/source_user.rb:4]: Why did you delete this?

Comment (rchubrich) [inline lib/twitter/source_user.rb:4]: Because it's awesomer that way, duh.
